xtask/src/bootstrap_groth16.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{fs, process::Command};
+
+use clap::Parser;
+use regex::Regex;
+
+#[derive(Parser)]
+pub struct BootstrapGroth16;
+
+const SOLIDITY_GROTH16_VERIFIER_PATH: &str =
+    "bonsai/ethereum/contracts/groth16/Groth16Verifier.sol";
+const RUST_GROTH16_VERIFIER_PATH: &str = "risc0/zkvm/src/host/groth16.rs";
+
+impl BootstrapGroth16 {
+    pub fn run(&self) {
+        let solidity_code = fs::read_to_string(SOLIDITY_GROTH16_VERIFIER_PATH).expect(
+            format!(
+                "failed to read the Solidity verifier from {}",
+                SOLIDITY_GROTH16_VERIFIER_PATH
+            )
+            .as_str(),
+        );
+        let mut rust_code = fs::read_to_string(RUST_GROTH16_VERIFIER_PATH).expect(
+            format!(
+                "failed to read groth16.rs from {}",
+                RUST_GROTH16_VERIFIER_PATH
+            )
+            .as_str(),
+        );
```

Comment:
It's possible to simplify `.as_ref()` (to keep consistency with lines 58 and 68 for example).

```suggestion
        let solidity_code = fs::read_to_string(SOLIDITY_GROTH16_VERIFIER_PATH).expect(
            &format!(
                "failed to read the Solidity verifier from {}",
                SOLIDITY_GROTH16_VERIFIER_PATH
            )
        );
        let mut rust_code = fs::read_to_string(RUST_GROTH16_VERIFIER_PATH).expect(
            &format!(
                "failed to read groth16.rs from {}",
                RUST_GROTH16_VERIFIER_PATH
            )
        );
```